README.md

```diff
@@ -1 +1,41 @@
-# data-management
+# Research Data Management
+
+This repo provides the documentation for the research data management processes at the [Quantitative Biology Center](https://ror.org/00v34f693) [<img src="https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg" height=18>](https://ror.org/00v34f693), [University of Tübingen](https://ror.org/03a1kwz48)[Quantitative Biology Center](https://ror.org/00v34f693) [<img src="https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg" height=18>](https://ror.org/00v34f693).
```

Comment:
```suggestion
This repo provides the documentation for the research data management processes at the [Quantitative Biology Center](https://ror.org/00v34f693) [<img src="https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg" height=18>](https://ror.org/00v34f693), [University of Tübingen](https://ror.org/03a1kwz48) [Quantitative Biology Center](https://ror.org/00v34f693) [<img src="https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg" height=18>](https://ror.org/00v34f693).
```
NIT: Without a space it looks cluttered:
![image](https://github.com/qbicsoftware/research-data-management/assets/29627977/1cfa1a9f-9ca8-44c7-859d-de5f6d9a1f75)